config/crd/patches/cainjection_in_monitoringconfigs.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
+  name: monitoringconfigs.everest.percona.com
```

Comment:
Please add a new CRD to the cleanup step in the pipeline https://github.com/percona/everest-operator/blob/main/.github/workflows/ci.yml#L155